packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -491,6 +495,9 @@ export class DocumentsSchemaController {
 					},
 			  } satisfies IDocumentSchemaCurrent);
 
+		checkRuntimeCompatibility(this.documentSchema, "document");
+		this.validateSeqNumber(this.documentSchema.refSeq, snapshotSequenceNumber, "summary");
```

Comment:
This validates that the document schema's seq# is <= the sequence number of the summary this container loaded from right? What does that mean?

Reply:
Yes, that's what I was asking. Thanks for explaning. Maybe you can add a comment for that.